### DISCUSSION TITLE
Mario 64

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#321] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/mario-64/

**Comment by brunoaugusto** [comment]
Só eu achei que esse "Também te amo, Peach" soou mais como "Também amo bolo de morango, Peach!" :rofl:

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/mario-64/#comment_1128942923>):

> Só eu achei que esse "Também te amo, Peach" soou mais como "Também amo bolo de morango, Peach!" 🤣

:rofl:
